app/controllers/articles_controller.rb

```diff
@@ -1,7 +1,25 @@
 class ArticlesController < ApplicationController
-
   def index
     @articles = Article.all
   end
 
+  def new
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+    if @article.save
+      flash[:notice] = "Article successfully saved and sent for approval"
+      redirect_to root_path
+    else
+      render 'new'
```

Comment:
Missing an error message if somethings goes wrong

Reply:
Also you need to write a sad path if the user enters a empty field, you have validations in place that content or title cant be blank.